src/scss/_utilities.scss

```diff
@@ -0,0 +1,106 @@
+@import 'variables';
+@import 'values';
+
+/// Traverse maps and retrieve deeply nested values: https://css-tricks.com/snippets/sass/deep-getset-maps/
+/// @author Hugo Giraudel
+/// @param {Map} $map A sass map and any number of keys.
+/// @param {*} $keys Nested values.
+/// @return Nested values or nested map.
+@function map-deep-get($map, $keys...) {
+  @each $key in $keys {
+    $map: map-get($map, $key);
+  }
+  @return $map;
+}
+
+/// Function returns the value for a given breakpoint.
+/// @param {String} $breakpointName Breakpoint name (sm, md, lg, xlg, max).
+/// @return {Length} The breakpoint value in rems.
+@function getBreakpointValue($breakpointName) {
```

Comment:
- [ ] I think @diego-codes wants us to move to kebab-case on these

Reply:
It is okay if you want to knock it out in the same one. I changed the base of the PR to v1.1.0.

Reply:
So you'd like to see me update everything in `functions.scss` & `mixins.scss`, or just in this new `utilities.scss` file? (If the latter, I worry about there still being inconsistency, as I'm using variables like `$allBreakpoints` in camel case, which come from other scss files).

Reply:
I just mean function and mixin names 😄